### PR TITLE
v2

### PR DESCRIPTION
**Pull request by simonvadee**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (LaRiffle) [inline client/public/index.html]: What is this for?

File: .github/workflows/server-v2.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node modules
+        uses: actions/cache@v1
+        with:
+          path: server-v2/node_modules
```
Comment (LaRiffle) [inline .github/workflows/server-v2.yml]: Change path?

File: client/.env [hunk 3]
```diff
@@ -0,0 +1,4 @@
+REACT_APP_ENGINE_BACKEND_URL=https://engine.arkhn.org
+REACT_APP_HTTP_BACKEND_URL=http://localhost:4000
+REACT_APP_WS_BACKEND_URL=ws://localhost:4000
+REACT_APP_CLEANING_SCRIPTS_URI=http://localhost:5000
```
Comment (LaRiffle) [inline client/.env]: Will be changed to 5001 (conflict port with fhir api)

File: client/src/components/Navbar/index.tsx [hunk 4]
```diff
@@ -1,83 +1,43 @@
...
+import useReactRouter from 'use-react-router';
 
-import Drawer from "./components/Drawer";
-import Header from "./components/Header";
+import Drawer from './drawer';
+import Header from './header';
 
-import { changeNode } from "../../services/selectedNode/actions";
-import { login, logout } from "../../services/user/actions";
+import { logout } from '../../services/user/actions';
```
Comment (LaRiffle) [inline client/src/components/Navbar/index.tsx]: Can we put absolute path?
`import { logout } from 'src/services/user/actions';`

File: client/src/components/authentication/index.tsx [hunk 5]
```diff
@@ -1,35 +1,32 @@
...
+import { Mutation } from 'react-apollo';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { loader } from 'graphql.macro';
 
-import Navbar from "../../components/Navbar";
+import Navbar from '../navbar';
 
-import { login as loginAction } from "../../services/user/actions";
+import { login as loginAction } from '../../services/user/actions';
```
Comment (LaRiffle) [inline client/src/components/authentication/index.tsx]: same suggestion to use absolute path when possible

File: client/public/index.html (hunk 1 above)
Comment (simonvadee) [reply client/public/index.html]: don't know, probably useless but it was there before

File: .github/workflows/server-v2.yml (hunk 2 above)
Comment (simonvadee) [reply .github/workflows/server-v2.yml]: done!

File: client/.env (hunk 3 above)
Comment (simonvadee) [reply client/.env]: thanks, done.

File: client/src/components/Navbar/index.tsx (hunk 4 above)
Comment (simonvadee) [reply client/src/components/Navbar/index.tsx]: done.

File: client/src/components/authentication/index.tsx (hunk 5 above)
Comment (simonvadee) [reply client/src/components/authentication/index.tsx]: done.

**Pull request merged**